src/pages/messages/components/TicketList.tsx

```diff
@@ -38,13 +40,22 @@ export const TicketList = ({
     }
   }, [tickets]);
 
+  const location = useLocation();
+  const currentPath = location.pathname.replace(/\/$/, '').toLowerCase(); // Remove trailing slash
+
   return (
     <section>
       <Helmet>
         <title>{helmetTitle}</title>
       </Helmet>
 
-      <SearchForm sx={{ mb: '1rem' }} placeholder={t('tasks.search_placeholder')} />
+      <SearchForm sx={{ mb: '1.8rem' }} placeholder={t('tasks.search_placeholder')} />
+
+      {currentPath === UrlPathTemplate.MyPageMyMessages ? (
+        <Typography variant="h2" sx={{ mb: '1rem' }}>
+          {t('common.dialogue')}
+        </Typography>
+      ) : null}
```

Comment:
Siden dette er en heading som skal beskrive innholdet under burde den være øverst på siden. Kan gjenbruke title fra `helmetTitle` også, siden de er like. Antar at samme endringen skal gjøres for Oppgaver-siden også

```suggestion
      <Typography variant="h2" sx={{ mb: '1rem' }}>
        {helmetTitle}
      </Typography>
      
      <SearchForm sx={{ mb: '1rem' }} placeholder={t('tasks.search_placeholder')} />
```